admin_manual/installation/nginx_configuration.rst

```diff
@@ -108,6 +106,108 @@ Nginx Configuration
 
     }
 
+Serving at https(s)://example.com/owncloud
+------------------------------------------
+
+.. note:: Assumes php-fpm; :ref:`using_php-fpm`
+
+.. code-block:: python
+
+
+    server {
+        listen 80;
+        server_name example.com;
+        # enforce https
+        return 301 https://$server_name$request_uri;
+    }
+
+    server {
+        listen 443 ssl;
+        server_name example.com;
+
+        ssl_certificate /etc/ssl/nginx/cloud.example.com.crt;
+        ssl_certificate_key /etc/ssl/nginx/cloud.example.com.key;
+
+        # Add headers to serve security related headers
+        add_header Strict-Transport-Security "max-age=0; includeSubDomains; preload;";
+        add_header X-Content-Type-Options nosniff;
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        add_header X-Robots-Tag none;
+
+        # Path to the root of your website (one level above owncloud folder)
+        root /var/www;
+        # set max upload size
+        client_max_body_size 10G;
+        fastcgi_buffers 64 4K;
+
+        # Disable gzip to avoid the removal of the ETag header
+        gzip off;
+
+        # Uncomment if your server is build with the ngx_pagespeed module
+        # This module is currently not supported.
+        #pagespeed off;
+
+        # ownCloud blacklist
+        location ~ ^/owncloud/(?:\.htaccess|data|config|db_structure\.xml|README) {
+            deny all;
+            error_page 403 = /owncloud/core/templates/403.php;
+        }
+
+        index index.php;
+
+        location = /robots.txt {
+            allow all;
+            log_not_found off;
+            access_log off;
+        }
+
+        location ~ ^/(?:\.htaccess|data|config|db_structure\.xml|README){
+            deny all;
+        }
+
+        location /owncloud {
+            error_page 403 = /owncloud/core/templates/403.php;
+            error_page 404 = /owncloud/core/templates/404.php;
+
+            rewrite ^/owncloud/caldav(.*)$ /remote.php/caldav$1 redirect;
+            rewrite ^/owncloud/carddav(.*)$ /remote.php/carddav$1 redirect;
+            rewrite ^/owncloud/webdav(.*)$ /remote.php/webdav$1 redirect;
+
+            rewrite ^(/owncloud/core/doc[^\/]+/)$ $1/index.html;
+
+            # The following rules are only needed with webfinger
```

Comment:
Those won't work because they need to be found at the root of the server like:

http://127.0.0.1/.well-known/carddav

instead of the current:

http://127.0.0.1/owncloud/.well-known/carddav

according to https://tools.ietf.org/html/rfc5785 / https://tools.ietf.org/html/rfc5785#appendix-B